### PR TITLE
Master: merge into master as demo

### PR DESCRIPTION
**Pull request by wmzspace**
Version: PreDemo 1.0

### CODE REVIEW

**Comment by wmzspace** [comment]
Please review and approve this pull request❤

**Review by wmzspace — COMMENTED** [review]
test

**Review by cheeterLee — APPROVED** [review]

File: test.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+console.log("We are coding gods!")
+print("We can do a good job!")
```
Comment (zhaojiason) [inline test.js:2]: Well done! I knew the basic idea of how to collaborate in GitHub. 
And have a good night, bros!

**Review by zhaojiason — APPROVED** [review]
Well done! I knew the basic idea of how to collaborate in GitHub.
And have a good night, bros!

File: test.js (hunk 1 above)
Comment (wmzspace) [reply test.js:2]: Good job bro! Have a good night!

**Pull request merged**